diff --git a/node_modules/next/dist/client/components/router-reducer/fetch-server-response.js b/node_modules/next/dist/client/components/router-reducer/fetch-server-response.js
index 653b8ce..709897a 100644
--- a/node_modules/next/dist/client/components/router-reducer/fetch-server-response.js
+++ b/node_modules/next/dist/client/components/router-reducer/fetch-server-response.js
@@ -95,7 +95,9 @@ async function fetchServerResponse(url, options) {
                 // between HTML and RSC requests. Instead, we append an extra prefix
                 // to the request.
                 url = new URL(url);
-                if (url.pathname.endsWith('/')) {
+                if (url.pathname === process.env.__NEXT_ROUTER_BASEPATH) {
+                    url.pathname += "/index.txt";
+                } else if (url.pathname.endsWith("/")) {
                     url.pathname += 'index.txt';
                 } else {
                     url.pathname += '.txt';
diff --git a/node_modules/next/dist/server/lib/router-utils/setup-dev-bundler.js b/node_modules/next/dist/server/lib/router-utils/setup-dev-bundler.js
index 6c205ba..366565b 100644
--- a/node_modules/next/dist/server/lib/router-utils/setup-dev-bundler.js
+++ b/node_modules/next/dist/server/lib/router-utils/setup-dev-bundler.js
@@ -288,9 +288,8 @@ async function startWatcher(opts) {
             let tsconfigChange = false;
             let conflictingPageChange = 0;
             let hasRootAppNotFound = false;
-            const { appFiles, pageFiles } = opts.fsChecker;
-            appFiles.clear();
-            pageFiles.clear();
+            const appFilesList = [];
+            const pageFilesList = [];
             _shared1.devPageFiles.clear();
             const sortedKnownFiles = [
                 ...knownFiles.keys()
@@ -433,7 +432,7 @@ async function startWatcher(opts) {
                     }
                     appPaths[pageName].push(opts.turbo ? originalPageName.replace(/%5F/g, '_') : originalPageName);
                     if (useFileSystemPublicRoutes) {
-                        appFiles.add(pageName);
+                        appFilesList.push(pageName);
                     }
                     if (validFileMatcher.isAppRouterRoute(fileName)) {
                         appRouteHandlers.push({
@@ -451,7 +450,7 @@ async function startWatcher(opts) {
                     }
                 } else {
                     if (useFileSystemPublicRoutes) {
-                        pageFiles.add(pageName);
+                        pageFilesList.push(pageName);
                         // always add to nextDataRoutes for now but in future only add
                         // entries that actually use getStaticProps/getServerSideProps
                         opts.fsChecker.nextDataRoutes.add(pageName);
@@ -488,6 +487,11 @@ async function startWatcher(opts) {
                 }
                 routedPages.push(pageName);
             }
+            const { appFiles, pageFiles } = opts.fsChecker;
+            appFiles.clear();
+            pageFiles.clear();
+            appFilesList.forEach((file) => appFiles.add(file));
+            pageFilesList.forEach((file) => pageFiles.add(file));
             const numConflicting = conflictingAppPagePaths.size;
             conflictingPageChange = numConflicting - previousConflictingPagePaths.size;
             if (conflictingPageChange !== 0) {
